code/lib/cli/src/automigrate/fixes/vite-config-file.ts

```diff
@@ -45,7 +45,16 @@ export const viteConfigFile = {
 
     const rendererName = frameworkToRenderer[frameworkName as keyof typeof frameworkToRenderer];
 
-    if (!viteConfigPath && isUsingViteBuilder) {
+    if (
+      !isViteConfigFileFound &&
+      mainConfig.core?.builder &&
+      typeof mainConfig.core?.builder !== 'string' &&
+      mainConfig.core?.builder.options
+    ) {
+      isViteConfigFileFound = !!mainConfig.core?.builder.options.viteConfigPath;
```

Comment:
couldn't this be simplified a little by just doing optional-chaining all the way down?
```suggestion
    ) {
      isViteConfigFileFound = !!mainConfig.core?.builder?.options?.viteConfigPath;
```

Reply:
Could be. Although I personally like the explicit style here a bit more because it makes clear, that `builder` can be a string or object.